src/Altinn.App.Core/Internal/AppModel/DefaultAppModel.cs

```diff
@@ -3,24 +3,30 @@
 
 namespace Altinn.App.Core.Internal.AppModel;
 
+/// <inheritdoc />
 public class DefaultAppModel: IAppModel
 {
     private readonly ILogger<DefaultAppModel> _logger;
     
+    /// <summary>
+    /// Create with services from Dependency Injcection
+    /// </summary>
     public DefaultAppModel(ILogger<DefaultAppModel> logger)
     {
         _logger = logger;
     }
     
+    /// <inheritdoc />
     public object Create(string classRef)
     {
         _logger.LogInformation($"CreateNewAppModel {classRef}");
-        return Activator.CreateInstance(GetModelType(classRef));
+        return Activator.CreateInstance(GetModelType(classRef))!;
     }
 
+    /// <inheritdoc />
     public Type GetModelType(string classRef)
     {
         _logger.LogInformation($"GetAppModelType {classRef}");
-        return Assembly.GetEntryAssembly().GetType(classRef);
+        return Assembly.GetEntryAssembly()?.GetType(classRef) ?? throw new Exception("App model of type {classRef} does not exist");
```

Comment:
```suggestion
        return Assembly.GetEntryAssembly()?.GetType(classRef, true);
```

Reply:
I'm generally in favor of throwing my own exceptions that explain what happens (especially when the error is triggered by a config error), but for your approach to work, we at least need to suppress the warnings with `!`
```suggestion
        return Assembly.GetEntryAssembly()!.GetType(classRef, true)!;
```

Reply:
I agree that throwing explicit exceptions is the best, but in this case GetType(classRef, true) throws a very explicit exception:
```
System.TypeLoadException: Could not load type 'Altinn.App.Models.Skjemas' from assembly 'Altinn.App, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
         at System.Reflection.RuntimeAssembly.GetType(QCallAssembly assembly, String name, Boolean throwOnError, Boolean ignoreCase, ObjectHandleOnStack type, ObjectHandleOnStack keepAlive, ObjectHandleOnStack assemblyLoadContext)
         at System.Reflection.RuntimeAssembly.GetType(String name, Boolean throwOnError, Boolean ignoreCase)
         at System.Reflection.Assembly.GetType(String name, Boolean throwOnError)
```
I think we don't need to handle cases where EntryAssembly is null, if we do we should throw an exception that explains that the EntryAssembly is null.
So I agree on your suggestion to my suggestion :)

Reply:
Yea, that message was pretty good.